data/map.txt

```diff
@@ -1999,6 +1998,63 @@ system Achernar
 			distance 307
 			period 23.907
 
+system Acrab
+	pos -297.08377 146.935
+	government Syndicate
+	link Fomalhaut
+	link Markab
+	link Tabit
+	asteroids "small rock" 40 4.224
+	asteroids "large rock" 32 6.27
+	asteroids "medium metal" 9 5.61
+	asteroids "large metal" 4 4.158
+	minables aluminum 3 10.167916
+	minables iron 7 9.6900905
+	fleet "Large Core Merchants" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
+	habitable 151.25
+	belt 1453
+	object
+		sprite star/m8
+		period 10
+	object
+		sprite planet/rock5-b
+		distance 255
+		period 296.14716
+	object
+		sprite planet/browndwarf-t-rouge
```

Comment:
In this extremely unusual case, the M8 may not actually be much brighter than the brown dwarf, so we might make an exception, but it still seems wrong to have it outside the orbit of this planet.

```suggestion
		sprite planet/browndwarf-t-rouge
		distance 255
		period 296.14716
	object
		sprite planet/rock5-b
```

Reply:
There is quite a bit of distance between them. The idea was that there's one planet that's close enough to the main star that its orbit can stably endure passing between the binary.
If the planet sprite needs to be changed though that's fair.